config/routes.rb

```diff
@@ -1,3 +1,19 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root to: 'pages#home'
+
+
+
+    resources :events do  sponsors
+                          companies
+                          workshops do users
+                          end
```

Comment:
I'm not sure. It doesn't make sense for the changes to the routes to be in the same pull request as the upcoming events view changes. I propose that I remove the routes changes from that pull request, and this one can focus on just the routes. Does that sound okay?